scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala

```diff
@@ -80,5 +80,5 @@ final class AvroScioContext(private val self: ScioContext) extends AnyVal {
 
 /** Enhanced with Avro methods. */
 trait ScioContextSyntax {
-  implicit def toAvroScioContext(c: ScioContext): AvroScioContext = new AvroScioContext(c)
+  implicit def toScioContextOps(c: ScioContext): ScioContextOps = new ScioContextOps(c)
```

Comment:
Would that not break the implicit conversion if `BigQueryScioContext` and `AvroScioContext` have the implicit conversion with the same name that are  both in scope ?

Reply:
@jto 🤦‍♂️ yes! 💣

Reply:
@jto I'll try to go with this convention `def <main package><ops class>( ... )`

Reply:
sounds good